client/src/pages/Configurations/ConfigurationTitleBar.tsx

```diff
@@ -7,27 +7,38 @@ type ConfigurationSteps = 'build' | 'test' | 'activate';
 
 type ConfigurationTitleBarProps = {
   step: ConfigurationSteps;
+  condition: string;
 };
 
-const CONFIGURATION_TITLE_CONTENTS = {
-  build: {
-    title: 'Build configuration',
-    subtitle: 'Review and select the data to retain in the eCRs you receive.',
-  },
-  test: {
-    title: 'Test configuration',
-    subtitle: 'Check the results of your configuration before turning it on.',
-  },
-  activate: {
-    title: 'Turn on configuration',
-    subtitle:
-      'Refiner will immediately start to refine eCRs with gonorrhea as a reportable condition, in accordance with this configuration.',
-  },
+type ConfigurationTitleContent = {
+  title: string;
+  subtitle: string;
+  boldedWords?: string;
```

Comment:
Mentioned in my other comment that this might be a good way to go!
```suggestion
  subtitle: React.ReactNode;
```